components/forms/Input/react.js

```diff
@@ -7,43 +7,55 @@ import style from './style.module.scss'
 const cx = classNames.bind(style)
 
 const Input = ({
-  error, onChange, onKeyDown, placeholder, value, variant,
-}) => (
-  <div className={cx('input-wrapper')}>
-    <input
-      className={cx('input', {
-        'input--light': variant === 'light',
-        'input--dark': variant === 'dark',
-        'input--error': error,
+  error, label, onChange, onKeyDown, placeholder, value, variant,
+}) => {
+  const [isActive, setActive] = useState(false)
+  const Label = label ? 'label' : 'div'
```

Comment:
We shouldn't need to fallback to a div since every input needs a label to describe the purpose of the form control.

If all of these inputs need a label, maybe we can add a prop to identify if a label is intended for screen reader only. If true, we can apply a screen reader class to the label making it visually hidden.